content/blog/2022-05-06-end-to-end-computer-vision-api-part-3-remote-exp-ci-cd.md

```diff
@@ -0,0 +1,292 @@
+---
+title:
+  'End-to-End Computer Vision API, Part 3: Remote Experiments & CI/CD For
+  Machine Learning'
+date: 2022-05-06
+description: >
+  This is the third and last part of this series of posts on how to build an
```

Comment:
this description is too long for the front page

Reply:
Change the description